indico/web/client/js/jquery/extensions/global.js

```diff
@@ -247,19 +247,31 @@ $(document).ready(function() {
   showFormErrors();
 
   // Show form creation dialog if hash is present in the URL
-  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d+))?$/);
+  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d*))(?::(\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b))?$/);
   if (match) {
     var eventType = match[1];
     var categoryId = match[2];
+    var eventUUID = match[3];
+    console.log("eventType: ", eventType)
+    console.log("categoryId: ", categoryId)
+    console.log("eventUUID: ", eventUUID)
     var title = {
       lecture: $T.gettext('Create new lecture'),
       meeting: $T.gettext('Create new meeting'),
       conference: $T.gettext('Create new conference'),
     }[eventType];
-    var url =
-      match[2] !== undefined
-        ? build_url(Indico.Urls.EventCreation, {event_type: eventType, category_id: categoryId})
-        : build_url(Indico.Urls.EventCreation, {event_type: eventType});
+    var url;
+    if (match[2] !== "") {
+      url = build_url(Indico.Urls.EventCreation, {event_type: eventType, category_id: categoryId});
+    } else if (match[3] !== "") {
+      url = build_url(Indico.Urls.EventCreation, {event_type: eventType, event_uuid: eventUUID});
+    } else {
+      url = build_url(Indico.Urls.EventCreation, {event_type: eventType});
+    }
```

Comment:
While not relevant for now I'd also support the case where we have both a category id and a uuid.

Reply:
Added a condition for when both category ID and a UUID are present. Does this need to be implemented now?

Reply:
I would just build the URL accordingly:

    url = build_url(Indico.Urls.EventCreation, {event_type: eventType, category_id: categoryId, event_uuid: eventUUID});

That way it will Just Work ™ if we ever have something passing data like this.